packages/profile-sync-controller/src/controllers/user-storage/UserStorageController.ts

```diff
@@ -45,6 +47,27 @@ import {
   upsertUserStorage,
 } from './services';
 
+// TODO: add external NetworkController event
```

Comment:
As noted, these are temporary.
We will create a separate PR that correctly adds and exposes this similar events.
If some of these events are not available, then we will attempt to use the global `:changeEvent`.